pkg/apis/serving/v1beta1/component.go

```diff
@@ -93,6 +94,9 @@ type ComponentExtensionSpec struct {
 	// Knative Pod Autoscaler(https://knative.dev/docs/serving/autoscaling/autoscaling-metrics).
 	// +optional
 	ScaleMetric *ScaleMetric `json:"scaleMetric,omitempty"`
+	// Type of metric to use. Options are Utilization, or AverageValue.
+	// +optional
+	ScaleMetricType *v2.MetricTargetType `json:"scaleMetricType,omitempty"`
```

Comment:
What's the difference between ScaleMetric and ScaleMetricType? seems like they are both metric types